backend/handlers/comment.go

```diff
@@ -0,0 +1,96 @@
+package handlers
```

Comment:
please refer to structure issues. Handlers should be in app dir
also name it as comment_handler